src/us49.py

```diff
@@ -0,0 +1,27 @@
+'''
+Author: Grace Miguel
+Date: 11/27/2020
+I pledge my honor that I've abided by the Stevens Honor Code.
+'''
+import sys
+import os
+#from Gedcom import Gedcom, Family
+from Gedcom import Gedcom, Family 
+#This function lists families with only daughters
+def us49(gedcom_file):
+    children = []
+    gedcom = Gedcom(gedcom_file)
+    families = gedcom.families
+    x = 0
+    for i in families:
```

Comment:
best practice to use follow a semantic naming scheme for the sake of readability. So this would be `for family in families`.

Reply:
fixed!